lagunita/lms/templates/su-header.html

```diff
@@ -0,0 +1,9 @@
+## mako
+
+<div id="su-header">
+  <div class="su-container">
+    <a href="http://www.stanford.edu">
+      <img src="https://www.stanford.edu/su-identity/images/brandbar-stanford-logo@2x.png" alt="Stanford University" width="152" height="23" />
```

Comment:
I think we should consider hosting this file directly from the platform.
I've noticed issues loading it in some places, like devstack,
maybe due to a `http` vs `https` issue.

Reply:
And just generally speaking, I'd prefer we host all of our own images.
If someone else's server breaks, I don't want our site degrading.